rdwatch/core/views/model_run.py

```diff
@@ -306,45 +289,31 @@ def paginate_queryset(
         filters: ModelRunFilterSchema,
         **params,
     ) -> dict[str, Any]:
-        # TODO: remove caching after model runs endpoint is
-        # refactored to be more efficient.
-        model_runs = None
-        cache_key = _get_model_runs_cache_key(filters.dict() | pagination.dict())
-
-        # If we have a cache miss, execute the query and save the results to cache
-        # before returning.
-        if model_runs is None:
-            qs = super().paginate_queryset(queryset, pagination, **params)
-            aggregate_kwargs = {
-                'timerange': JSONObject(
-                    min=ExtractEpoch(Min('evaluations__start_date')),
-                    max=ExtractEpoch(Max('evaluations__end_date')),
-                ),
-            }
-            if filters.region:
-                aggregate_kwargs['bbox'] = Coalesce(
-                    BoundingBoxGeoJSON('region__geom'),
-                    BoundingBoxGeoJSON('evaluations__geom'),
-                )
-
-            model_runs = qs | queryset.aggregate(**aggregate_kwargs)
-            if filters.region:
-                if qs['count'] == 0:  # No model runs we set bbox to Region bbox
-                    region_filter = filters.dict()['region']
-                    regions = [
-                        obj
-                        for obj in Region.objects.all()
-                        if obj.value == region_filter
-                    ]
-                    if len(regions) > 0:
-                        bbox = regions[0].geojson
-                        model_runs['bbox'] = bbox
-            cache.set(
-                key=cache_key,
-                value=model_runs,
-                timeout=timedelta(days=30).total_seconds(),
+        qs = super().paginate_queryset(queryset, pagination, **params)
+        aggregate_kwargs = {
+            'timerange': JSONObject(
+                min=ExtractEpoch(Min('evaluations__start_date')),
+                max=ExtractEpoch(Max('evaluations__end_date')),
+            ),
+        }
+        if filters.region:
+            aggregate_kwargs['bbox'] = Coalesce(
+                BoundingBoxGeoJSON('region__geom'),
+                BoundingBoxGeoJSON('evaluations__geom'),
             )
 
+        model_runs = qs | queryset.aggregate(**aggregate_kwargs)
+        if filters.region:
+            if qs['count'] == 0:  # No model runs; we set bbox to Region bbox
+                region_filter = filters.dict()['region']
+                # We do a full table scan because this isn't a common case and isn't a bottleneck;
+                # the equivalent database query is really annoying due to the logic of the
+                # `Region.value` property.
+                for region in Region.objects.all():
+                    if region.value == region_filter:
+                        model_runs['bbox'] = region.geojson
```

Comment:
I think we _might_ be able to replace the `Region.value` property with a `GeneratedField` to get around this. But, I think that's fine to postpone for the future

Reply:
Yeah, this logic existed prior to my change and isn't really related, but I wanted to add a comment about it since it looks suspicious.